File: Programmers/Level1/같은숫자는싫어.cpp
#include <vector>
#include <iostream>

using namespace std;

vector<int> solution(vector<int> arr)
{
    vector<int> answer;

    for(int i = 0 ; i < arr.size()-1 ; i ++){
      if(i == 0 ){
        answer.push_back(arr[i]);
      }
      
      if( arr[i] != arr[i+1]){
        answer.push_back(arr[i+1]);
      }
    }

    return answer;
}

File: JavaScript/[1]Programmers/코딩테스트 고득점 Kit/같은숫자는싫어.js
function solution(arr) {

    const answer = [];

    arr.forEach( (value, index, array) => {
        if (index == 0) {
            answer.push(value);
        } else if (array[index-1] !== value){
            answer.push(value);
        }
    });
    
    return answer;
}

const arr1 = [1,1,3,3,0,1,1];
const arr2 = [4,4,4,3,3];

console.log(solution(arr1));

